### ISSUE TITLE
Display Room preview when edit heightmap

### ISSUE THREAD

**Issue by odrevet** [lordmir/landstalker_editor#67]
This is sometime difficule to know which height is aligned to the tilemap. A preview of the room, when the Heightmap page is open in the Room notebook,  would make easier to edit heightmap. 

Similar to the Room page with heightmap display on, with the focused heightmap tile also highlighted in the preview. 

Should this additional widget be added using wxcrafter or by coding it directly ? It seems that `src/user_interface/wxresource/resource/wxcrafter.wxcp` has not been updated with the latest additions

**Comment by lordmir** [comment]
There is a way of previewing the room whilst editing the heightmap - you can drag the room view tab to the left or right to create a side-by-side view. This will give you something similar to the following:

![image](https://github.com/user-attachments/assets/d3098230-de6b-4791-b6e4-55b330158d67)

As this isn't an ideal solution, and having a preview built-in to the editor would be a lot more useful, I'll keep this issue open. But the above workaround will work for now (minus the hovered heightmap cell)

**Comment by odrevet** [comment]
I did not know that the views can be split. This is a little better that the separated view. 
I was also thinking instead of a preview, to have an unified view: 

Remove the notebook and only keep the Room page. 
Then having an "edit mode" choice with entities /  heightmap / background / foreground that will mask or alpha dim layers and draw the wire map and info that was previously on each page

**Comment by lordmir** [comment]
I agree a unified view would be better, and is part of a general set of improvements I would like to make to the room editor. This would also include using some hardware acceleration in order to improve the performance of the editor, as it is currently quite slow and unresponsive. Any additional drawing using wx/OS GUI commands (especially bitmaps) would only slow things down further, hence wanting to use the GPU instead.
This will take some time to implement, as it isn’t an area I'm fully familiar with. At the moment, I'm focusing on adding support for editing the remaining game assets.
